backend/src/main/java/com/javahelp/backend/data/ISurveyStore.java

```diff
@@ -0,0 +1,40 @@
+package com.javahelp.backend.data;
+
+import com.amazonaws.regions.Regions;
+import com.javahelp.model.survey.Survey;
+
+public interface ISurveyStore {
```

Comment:
Do we need this to extend the `ICRUDGateway` interface? (I think Jacob and you had this conversation before but I don't remember the specifics)

Reply:
I extended ISurveyResponse but not SurveyStore. Not the most urgent thing to change but will change it